R/render-cached-plot.R

```diff
@@ -293,6 +293,7 @@ renderCachedPlot <- function(expr,
   cacheKeyExpr,
   sizePolicy = sizeGrowthRatio(width = 400, height = 400, growthRate = 1.2),
   res = 72,
+  alt = "Plot object",
```

Comment:
Maybe we could be a bit more descriptive here?

```suggestion
  alt = "A plot image generated by R Shiny",
```

Reply:
Thanks very much for the great suggestion, Carson! The reason why I have simplified it as "Plot object" is because I thought that was the minimal nitty-gritty. Users would know the plot has been generated by Shiny if they came  across the plot object within Shiny apps. Even it is not the case, I think shorter alt text would be preferred; otherwise, they would need to consume extra seconds to listen to decorative info. :)

Reply:
Ok, sounds good to me!